client/src/components/Allocations/AllocationsModals/AllocationsManageTeamModal/AllocationsManageTeamModal.js

```diff
@@ -181,6 +193,9 @@ const AllocationsManageTeamModal = ({ isOpen, toggle, projectId }) => {
           )}
         </div>
       </ModalBody>
+      <ModalFooter>
+        <Button styleName="update-button">Close</Button>
+      </ModalFooter>
```

Comment:
I think the [design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/505e3bec-059e-4bc0-97a1-d514d7103811) includes some footer text. Can you add that?